### ISSUE TITLE
Option to open automagically generated random forwarding port in firewall 

### ISSUE THREAD

**Issue by j00p34** [MissionCriticalCloud/vagrant-cloudstack#109]
When generating random ports for forwarding we need to open a range in the firewall. It would be very nice if we could have the option to open the random port only.

**Comment by bheuvel** [comment]
@j00p34 , if you set:

```
    p.pf_open_firewall      = "true"
    p.pf_trusted_networks   = trusted_networks
```

it will open the firewall for the (automagically random generated or not) forwarding port.

trusted network can be 
e.g. multiple  `trusted_networks = "1.2.3.4/24,6.7.8.9/32"`
or even easier: `trusted_networks = "#{Net::HTTP.get(URI.parse('https://api.ipify.org')).chomp}/32"`
This last one will specify the public address you are currently working from.

Let me know if it doesn't work
(Implemented in PR #99 )

**Comment by j00p34** [comment]
ok
